src/test/validation_flush_tests.cpp

```diff
@@ -30,7 +30,7 @@ BOOST_AUTO_TEST_CASE(getcoinscachesizestate)
     // (prevector<28, unsigned char>) when assigned 56 bytes of data per above.
     //
     // See also: Coin::DynamicMemoryUsage().
-    constexpr unsigned int COIN_SIZE = is_64_bit ? 80 : 64;
+    constexpr size_t COIN_SIZE{64};
```

Comment:
This is a surprising change, can you please explain it?
Do we even get to `BOOST_CHECK_EQUAL(view.DynamicMemoryUsage(), is_64_bit ? 32U : 16U);` here - and if so, how come none of the other values need changing?

Reply:
[I've checked](https://github.com/bitcoin/bitcoin/pull/32244) and most of the test is dead for the [past 3 years](https://github.com/bitcoin/bitcoin/pull/25325), cc @martinus.

Could you update the modified test all the way to revive it and define the desired behavior?